packages/crayons-core/src/index.html

```diff
@@ -12,7 +12,43 @@
     <script nomodule src="/build/crayons.js"></script>
   </head>
   <body>
-    <fw-button expand color="primary">Primary Button</fw-button>
+<script>
```

Comment:
Kindly add this to the end of the file. Comment this out.

Reply:
Fixed ✅